test/test_helper.rb

```diff
@@ -8,5 +8,10 @@
 require "imgix"
 require "webmock/minitest"
 
+require 'minitest/autorun'
+require 'minitest/reporters' # requires the gem
```

Comment:
I don't think we need this comment here. Also, what is the purpose of requiring this?

Reply:
😆  yea IDK why the heck the import is duplicated and why I threw that comment in there 🤷🏼 . I'll go ahead and remove that duped import and the comment. Good catch and sry about that.

As for what that gem is doing, from the PR description:
> Using gem 'minitest-reporters', miniTest output logs now behave similarly to rSpec's, with color formatting and better diffing.